kolibri/core/assets/src/views/auth-message.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+
+  <div class="auth-message">
+    <h1>
+      {{ header || $tr('loginPrompt') }}
+    </h1>
+    <p>
+      {{ details || defaultDetails }}
+    </p>
+  </div>
+
+</template>
+
+
+<script>
+
+  const userRoles = [
+    'admin',
```

Comment:
I didn't see anything like that in the current code, but added a new deviceOwner role for that possibility.